src/component/user-name.js

```diff
@@ -41,7 +41,7 @@ class UserName extends Component {
 					<form className="user">
 						<TextField value={userName} autoFocus id="user-name-input" placeholder="User Name" style={{margiBbottom: '5px'}} onChange={this.handleChange} />
 						<Link replace to={{ pathname: '/chat', state: { userName: userName.length > 0 ? userName : null } }}>
-							<Button type='submit' color="primary" variant="raised" fullWidth onClick={this.handleClick}>Continue</Button>
+							<Button type='submit' color="primary" variant="contained" fullWidth onClick={this.handleClick}>Continue</Button>
```

Comment:
This too.

Reply:
> This too.

![Desktop 2019-07-08 00-11-13](https://user-images.githubusercontent.com/11846339/60772493-f5fe1c80-a114-11e9-975b-202a371c45a3.png)